acsoo/templates/project/+project.name+/.coveragerc.bob

```diff
@@ -0,0 +1,27 @@
+[paths]
+source =
+    odoo/addons
+    */{{{ project.name }}}/odoo/addons
+
+[run]
+dynamic_context = test_function
+
+[report]
+include =
+    */odoo/addons/*
+
+omit =
+    */scenario/*
+    */scenarios/*
+    */test/*
+    */tests/*
+    */lib/*
+    *_example/*
+    *__init__.py
+    *__manifest__.py
+    *hooks.py
+    *_parser.py
+
+exclude_lines =
+    pragma: no cover
+    pragma: no-cover
```

Comment:
In general I would avoid excluding or omitting too much stuff in the default config. So I would shrink this .coveragerc.bob to the strict minimum.

Reply:
We could keep only:

```
omit =
    */test/*
    */tests/*
    *__init__.py
    *__manifest__.py

exclude_lines =
    pragma: no-cover
```

Reply:
I would omit only manifests.
- Tests are supposed to be covered since they are executed. If they are not, there is dead code in tests ?
- `__init__.py` are covered too since they are run at module import time ?